modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/liveusers/messaging/LiveUsersMessageListener.java

```diff
@@ -1,22 +1,28 @@
 /**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.portal.liveusers.messaging;
+package com.liferay.site.admin.web.internal.liveusers.messaging;
```

Comment:
@JanisZhang Could you double check whether this should be in site.admin or user.admin modules?